src/app/app.js

```diff
@@ -1,101 +1,98 @@
-import { playTransition } from "./util/animate-transitions";
+import playTransition from './util/animate-transitions';
 
-export const run = () => {
+export default function run() {
```

Comment:
- [ ] Please change this to es6 arrow function syntax, Fix this and **other similar regular functions too**.